pkg/comp-functions/functions/vshnnextcloud/deploy.go

```diff
@@ -256,10 +256,20 @@ func newValues(ctx context.Context, svc *runtime.ServiceRuntime, comp *vshnv1.VS
 		}
 	}
 
+	if len(comp.Spec.Parameters.Service.FQDN) == 0 {
+		return nil, fmt.Errorf("FQDN array is empty, but requires at least one entry: %w", errors.New("empty fqdn"))
+	}
+
+	trustedDomain := []string{
+		comp.GetName() + "." + comp.GetInstanceNamespace() + ".svc.cluster.local",
+	}
+	trustedDomain = append(trustedDomain, comp.Spec.Parameters.Service.FQDN...)
```

Comment:
I'd make here verification, similar to: https://github.com/vshn/appcat/blob/master/pkg/comp-functions/functions/vshnnextcloud/collabora.go#L56

or even now I'm thinking that we could move this input verification to webhook, so we won't receive ticket to debug broken fqdn

Reply:
I've added the validation